x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/components/connection_status/connection_status.js

```diff
@@ -56,7 +56,7 @@ export function ConnectionStatus({ isConnected }) {
             {icon}
           </EuiFlexItem>
 
-          <EuiFlexItem>
+          <EuiFlexItem style={{ whiteSpace: 'nowrap' }}>
```

Comment:
It's not very well-documented, but it's a best practice for us to use classnames instead of inline styles. Could you create `components/connection_status/_index.scss` and create a class for this?

```scss
.remoteClustersConnectionStatus__message {
  white-space: nowrap;
}
```

You'll need to import this file from `public/index.scss`:

```scss
// At the top of the file
@import './sections/remote_cluster_list/components/connection_status/index'; // (note this can be index, not _index, because SCSS ignores the leading underscore in filenames)
```

Then we can apply the class like this:

```jsx
<EuiFlexItem className="remoteClustersConnectionStatus__message">
```

Reply:
ok!